xdsl/irdl.py

```diff
@@ -1005,9 +1006,17 @@ def wrong_field_exception(field_name: str) -> PyRDLOpDefinitionError:
                 continue
             if parent_cls is Operation:
```

Comment:
Can we write `parent_cls in Operation.mro()` to make it future proof? (Generic will have to be implemented separately)

Reply:
I'll give it a go